### PR TITLE
:recycle: Investment section refactor

### PR DESCRIPTION
**Pull request by rrortegaa**
- :recycle: Investment section in Javascript page, refactor in progress
- :recycle: Refactored Investment section in Javascript page

## PR Description

- **Refactored investment section into Javascript page**

### CODE REVIEW

**Review by arianaomi — COMMENTED** [review]
Screen Shot 2022-03-16 at 21 51 34

EL degradado debe ser más oscuro para que no se pierda la letra de las cards

**Review by devCharles — APPROVED** [review]

**Review by arianaomi — APPROVED** [review]
🚀

File: components/Pages/Bootcamps/Investment.tsx [hunk 1]
```diff
@@ -20,51 +22,72 @@ export interface Props {
...
-            <div className='description'>
-              <Text>
-                {investment.description}
-              </Text>
-            </div>
+    <div className={classNames(
+      'bg-javascript-investment-lg',
+      'bg-cover bg-center bg-no-repeat',
+      'w-full'
+    )}>
```
[the remark this answers is not in the document]
Comment (rrortegaa) [reply components/Pages/Bootcamps/Investment.tsx:31]: - No, porque pageSection tiene el gradiente

**Review by evcorona — APPROVED** [review]
👍

**Pull request merged**